doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,259 @@
+==================================
+Granuheap
+==================================
+
+This example simulates the flow of a cohesive granular material inside a Granuheap instrument using the discrete element method (DEM). For more detailed information on the concepts and physical meaning of the parameters in Lethe, we strongly recommend visiting `DEM parameters <../../../parameters/dem/dem.html>`_.
+
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles``
+- Three-dimensional problem
+- Moving solid object
+- `GMSH <https://gmsh.info/>`_ grids
+
+
+----------------------------
+Files Used in This Example
+----------------------------
+
+All files mentioned below are located in the example's folder (``examples/dem/3d-granuheap``).
+
+- Mesh file: ``cylinder.msh`` and ``support.msh``
+- Parameters file for the problem: ``granuheap.prm``
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the flow of water-wet sand during a Granuheap experiment. First, the packing of particles is generated by inserting particles inside a cylinder. After 1.6s, the cylinder rises, and the particles can flow freely, producing a heap. We compare the numerical results with our experimental results. To remain consistent with granuheap, we use the same metrics in our simulations: a cylinder of 0.01m in diameter. For more information about granuheap procedures, visit the `granuheap web page <https://www.granutools.com/en/granuheap>`_. To match our experimental results, we test multiple combinations of parameters in our simulations. We use the work of Hu *et al.* `[1] <https://doi.org/10.1016/j.powtec.2021.11.049>`_ as a basis for the choice of variable parameters.
```

Comment:
```suggestion
This example simulates the flow of wet sand during a Granuheap experiment. First, the packing of particles is generated by inserting particles inside a cylinder. After 1.6s, the cylinder rises, and the particles can flow freely, producing a heap. We compare the numerical results with our experimental results. To remain consistent with Granuheap, we use the same metrics in our simulations: a cylinder of 0.01m in diameter. For more information about granuheap procedures, visit the `granuheap web page <https://www.granutools.com/en/granuheap>`_. To match our experimental results, we test multiple combinations of parameters in our simulations. We use the work of Hu *et al.* `[1] <https://doi.org/10.1016/j.powtec.2021.11.049>`_ as a basis for the choice of variable parameters.
```

what do you mean for variable parameters? Maybe I would say here contact properties?